docs/content/preview/faq/comparisons/cassandra.md

```diff
@@ -58,13 +57,13 @@ Local secondary indexes in Apache Cassandra ([see blog](https://pantheon.io/blog
+Some further details on the document data modeling of both databases is warranted. Apache Cassandra's JSON support can be misleading for many developers. YCQL allows SELECT and INSERT statements to include the JSON keyword. The SELECT output will now be available in the JSON format and the INSERT inputs can now be specified in the JSON format. However, this "JSON" support is an ease-of-use abstraction in the CQL layer that the underlying database engine is unaware of. Because there is no native JSON data type in CQL, the schema doesn't have any knowledge of the JSON provided by the user. This means the schema definition doesn't change nor does the schema enforcement. Cassandra developers needing native JSON support previously had no choice but to add a new document database such as MongoDB or Couchbase into their data tier.
 
 With YugabyteDB's native JSON support, developers can now benefit from the structured query language of Cassandra and the document data modeling of MongoDB in a single database.
 
 ## Operational stability / Add node challenges
 
-1. Apache Cassandra's eventually-consistent core implies that a new node trying to join a cluster cannot simply copy data files from current "leader". A logical read (quorum read) across multiple surviving peers is needed with Apache Cassandra to build the correct copy of the data to bootstrap a new node with. These reads need to uncompress/recompress the data back too. With Yugabyte, a new node can be bootstrapped by simply copying already compressed data files from the leader of the corresponding shard.
+1. Apache Cassandra's eventually-consistent core implies that a new node trying to join a cluster cannot just copy data files from current "leader". A logical read (quorum read) across multiple surviving peers is needed with Apache Cassandra to build the correct copy of the data to bootstrap a new node with. These reads need to uncompress/recompress the data back as well. With Yugabyte, a new node can be bootstrapped by copying already compressed data files from the leader of the corresponding shard.
 
 1. Apache Cassandra is implemented in Java. GC pauses plus additional problems, especially when running with large heap sizes (RAM). YugabyteDB is implemented in C++.
```

Comment:
1. Apache Cassandra is implemented in Java. GC poses additional problems, especially when running with large heap sizes. YugabyteDB, however, is implemented in C++.